veros/settings.py

```diff
@@ -58,9 +58,9 @@
     ("enable_neutral_diffusion", Setting(False, bool, "enable isopycnal mixing")),
     ("enable_skew_diffusion", Setting(False, bool, "enable skew diffusion approach for eddy-driven velocities")),
     ("enable_TEM_friction", Setting(False, bool, "TEM approach for eddy-driven velocities")),
-    ("K_iso_0", Setting(0.0, float, "constant for isopycnal diffusivity in m^2/s")),
+    ("K_iso_0", Setting(1000.0, float, "constant for isopycnal diffusivity in m^2/s")),
```

Comment:
I would rather not change these default settings, since it breaks compatibility with PyOM. Is there any particular reason for this change?

Reply:
Is it for K_iso_0, K_gm_0 and kappaM_min? In case, one doesn't specify K_gm_0=1000 and switch off IDEMIX+EKE then simulation results must be wrong because K_gm_0 = 0.